Minor dialog changes.

IguanaBob opened the pull request:
Just thought these sounded a bit better.

IguanaBob committed:
Minor dialog changes.

IguanaBob committed:
Fix name

ViewtifulSlayer reviewed (changes requested):
This looks good, but don't forget that [N] is a line break and dialogue boxes can only hold 26 characters, and [NAM] tags are counted. Line 36 just needs that [N] tag as shown below and it's good:

`[TPL:23][TPL:E][DLY:0][NAM:9]: The moment is[N]nearly upon us. Everyone,[N]lend your power to [NAM:0]![END]`

ViewtifulSlayer committed:
Update chapter35.asm

Tested @IguanaBob's text adjustments, added a line break [N] tag to line 36, and approved the commit.
